python/py-discid/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-discid
+github.setup        JonnyJD python-discid 1.2.0 v
+github.tarball_from archive
+revision            0
+
+categories-append   audio devel
+license             LGPL-3+
+platforms           {darwin any}
+supported_archs     noarch
+
+description         Python bindings for libdiscid by MusicBrainz.
+long_description    The main purpose is the calculation of an identifier for \
+                    audio discs (Disc ID) to use for the MusicBrainz database. \
+                    Additionally the disc MCN and track ISRCs can be extracted.
+
+homepage            https://python-discid.readthedocs.io/
+
+maintainers         nomaintainer
+
+checksums           rmd160  dd8a71b5e1d06fbcafd8a9bd8de2921ea2bc1694 \
+                    sha256  4eec56f054d960b66c4e42c711c317e987dd50c9e7267a29f7adadd9ceee664f \
+                    size    30756
+
+python.versions     26 27 32 33 34 35 36 37 38 39 310 311 312 313
```

Comment:
Is there any use-case for having python 2.6 and 3.2–3.7?

Reply:
Just looks at recent commits in python dir, even 3.8 been thrown out. Personally I am indifferent here, I just think it makes little sense to have python versions which are effectively dropped by MacPorts.

Reply:
don't add all of this stuff. Since it's only a dependency of the `isrcsubmit` port you add here (which ideally would use Python 3.13), you should only add that here as well. If you think others are useful, please only Python 3.10+ by now.

Reply:
They have released a new version (1.3.0) which requires Python 3.9, so this is resolved anyway.